src/glTypes.jl

```diff
@@ -19,7 +19,7 @@ const GLhalf = Cushort
 const GLenum = Cuint
 const GLboolean = Cuchar
 const GLclampf = Cfloat
-const GLsizei = Cssize_t
+const GLsizei = Cint
```

Comment:
Those have different sizes.. Was this really incorrect before?

Reply:
Ah, sorry...Didn't see your comment... Wow! That's interesting :-O

Reply:
@SimonDanisch the errors seemed to be that any time i passed in a pointer to GLsizei, for OpenGL to tell me the length of something (e.x. a uniform name string, or an error log string), it would sometimes return a ridiculously large size, on the order of hundreds of billions.

It was extremely infrequent when getting uniform names, but happened every time when trying to read the error log.